### DISCUSSION TITLE
Worker 进程能否给自定义进程发送消息

### DISCUSSION THREAD

**Discussion by liuende501** [swoole/swoole-src#5059] [Q&A] [answered]
通过 addProcess 添加自定义进程后, 自定义进程的父进程是 Manager, 在另一个 worker 进程中能否发送消息给自定义消息

**Comment by NathanFreeman** [comment, marked as answer]
可以，详细看这里 https://wiki.swoole.com/#/server/methods?id=addprocess
主要是worker使用Swoole\Process的方法与自定义进程通信。

**Reply by liuende501** [reply]
你好, 文档我看过, process 是在第三包里定义的, 取不到 process 变量
我能否通过我现在的 worker 进程, 获取到 manager, 找到自定义的 process, 然后发送消息

**Reply by NathanFreeman** [reply]
你用的是哪个第三方包
